bookserver/routers/rslogging.py

```diff
@@ -71,3 +73,17 @@ async def log_book_event(entry: LogItemIncoming, request: Request):
         return {"status": "OK", "idx": idx}
     else:
         return {"status": "FAIL"}
+
+
+@router.post("/set_tz_offset")
+def set_tz_offset(
+    response: Response, tzreq: TimezoneRequest, RS_info: Optional[str] = Cookie(None)
+):
+    if RS_info:
+        values = json.loads(RS_info)
+    else:
+        values = {}
+    values["tz_offset"] = tzreq.timezoneoffset
```

Comment:
I'd suggest a `constants.py` file or something like that, where we put repeated strings such as `RS_info` and `tz_offset`.